src/vm/fcall.h

```diff
@@ -1157,6 +1158,7 @@ struct FCSigCheck {
 #define HCCALL5(funcname, a1, a2, a3, a4, a5)   funcname(a1, a2, a5, a4, a3)
 #define HCCALL1_PTR(rettype, funcptr, a1)        rettype (F_CALL_CONV * funcptr)(a1)
 #define HCCALL2_PTR(rettype, funcptr, a1, a2)    rettype (F_CALL_CONV * funcptr)(a1, a2)
+#define HCCALL2_TYPEDEF(rettype, ty, a1, a2)     typedef rettype (F_CALL_CONV * ty)(a1, a2)
```

Comment:
You can just use the existing `HCCALL2_PTR` macro for this - check what's done for the same problem in vm\gchelpers.h